File: app/src/main/java/com/tarambana/markit/DataContainers/MarkSchemePart.java
package com.tarambana.markit.DataContainers;

public class MarkSchemePart {

    @com.google.gson.annotations.SerializedName("id")
    private String mID;

    @com.google.gson.annotations.SerializedName("PartAssignmentID")
    public int partAssignmentID;

    @com.google.gson.annotations.SerializedName("PartSectionID")
    public int partSectionID;

    @com.google.gson.annotations.SerializedName("PartPartID")
    public int partID;

    public int getPartID() {
        return partID;
    }

    public void setPartID(int partID) {
        this.partID = partID;
    }

    @com.google.gson.annotations.SerializedName("PartName")
    public String partName;

    @com.google.gson.annotations.SerializedName("PartAvailableMarks")
    public int partAvailableMarks;

    @com.google.gson.annotations.SerializedName("PartAuthor")
    public String partAuthor;

    public int getPartSectionID() {
        return partSectionID;
    }

    public String getPartName() {
        return partName;
    }

    public int getPartAvailableMarks() {
        return partAvailableMarks;
    }

}

File: MarkItDesktop/MarkItDesktop/Data Containers/MarkSchemePart.cs
using System;

namespace MarkItDesktop.Data_Containers
{
    class MarkSchemePart
    {
        public String partAuthor { get; set; }

        public String partName { get; set; }

        public int partAvailableMarks { get; set; }
 
        public int partAssignmentID { get; set; }

        public int partSectionID { get; set; }

        public int partPartID { get; set; }

        public MarkSchemePart()
        {

        }
    }
}
